cmd/kubelet/app/options/options.go

```diff
@@ -56,11 +56,6 @@ type KubeletFlags struct {
 	KubeConfig          string
 	BootstrapKubeconfig string
 
-	// Insert a probability of random errors during calls to the master.
-	ChaosChance float64
-	// Crash immediately, rather than eating panics.
-	ReallyCrashForTesting bool
```

Comment:
@liggitt, @BenTheElder  do you know if any tests actually set --really-crash-for-testing?

Reply:
https://cs.k8s.io/?q=really-crash-for-testing&i=nope&files=&repos=

Doesn't look like it

Reply:
https://cs.k8s.io/?q=really-crash-for-testing&i=nope&files=&repos=

Doesn't look like it